app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt

```diff
@@ -91,7 +91,7 @@ class TimerFragment : Fragment() {
 
         removeButton.setOnClickListener {
             viewModel.removeTimer {
-                Toast.makeText(context, "最後のタイマーは削除できません.", Toast.LENGTH_SHORT).show()
+                Toast.makeText(context, getString(R.string.cannot_delete_last_timer), Toast.LENGTH_SHORT).show()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/eda77881d3fe18550b771a3b107ca15ca7faf5c3/app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt#L94"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>